src/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import './Header.css';
+import { useState, useEffect } from 'react';
+import {Route, Routes, Link} from 'react-router-dom';
+import {getTopics} from './utils/api.js'
+
+
+const Header = () => {
+    const [topics, setTopics] = useState([])
+    useEffect(() => {
+        getTopics()
+        .then((topics) => {
+            setTopics(topics)
+        })
```

Comment:
Same applies here with regards to the array as second argument

Reply:
Thanks - I fixed that last night, will be fixed on the next commit.